src/OrchardCore.Modules/OrchardCore.Html/Views/HtmlBodyPart.Edit.cshtml

```diff
@@ -39,6 +40,21 @@
                 mode: { name: "htmlmixed" }
             });
 
+            var html = document.documentElement;
```

Comment:
It just feels like we should have a shared imported function for this. But for this PR it is all good. Eventually this entire code will need to be moved to a javascript file.